javascript-functions/main.js

```diff
@@ -0,0 +1,35 @@
+function convertMinutestoSecond(minutes) {
+  var seconds = minutes * 60;
+  console.log(seconds);
+}
+convertMinutestoSecond(5);
+
+function greet(name) {
+  console.log('Hey,', name);
+}
+
+greet('Lamberto');
+
+function getArea(w, h) {
+  console.log(w * h);
+}
+
+getArea(420, 70);
+
+function getFirstName(person) {
+  person = {
```

Comment:
Instead of defining the person object here, you should be defining it when you call the function, and accessing it using the parameter